go-controller/pkg/util/node_annotations.go

```diff
@@ -54,26 +54,28 @@ const (
 )
 
 type L3GatewayConfig struct {
-	Mode           config.GatewayMode
-	ChassisID      string
-	InterfaceID    string
-	MACAddress     net.HardwareAddr
-	IPAddresses    []*net.IPNet
-	NextHops       []net.IP
-	NodePortEnable bool
-	VLANID         *uint
+	Mode                   config.GatewayMode
+	ChassisID              string
+	InterfaceID            string
+	MACAddress             net.HardwareAddr
+	IPAddresses            []*net.IPNet
+	NextHops               []net.IP
+	NodePortEnable         bool
+	DisableSNATExternalGWs bool
```

Comment:
You can't set `DisableSNATExternalGWs` differently per-node, so this should not be part of the gateway annotation. `gateway_init.go` should just look at `config.DisableSNATExternalGWs`

Reply:
done